globus_automate_client/flows_client.py

```diff
@@ -155,6 +167,11 @@ def validate_flow_definition(flow_definition: Mapping[str, Any]) -> None:
     return
 
 
+def _get_flows_base_url_for_environment():
+    environ = os.environ.get("GLOBUS_SDK_ENVIRONMENT")
+    return _ENVIRONMENT_FLOWS_BASE_URLS[environ]
```

Comment:
Can we be extra defensive here by doing a `get` that defaults to one of the production URL keys?

Reply:
I was defensive by making `None` one of the keys in `_ENVIRONMENT_FLOWS_BASE_URL` Perhaps that is a bit farther a way so harder to see, but it also seems defensive in a decent way? I'm fine changing if you have a preference.

Reply:
Ah, read the next comment, and it triggered what I think you were saying: users could provide any old value in their environment and that might fault here.

I guess that begs the question whether an unknown value should default to prod or whether it should throw an error. I guess if I was going for `sandbox` and typo-d it, I'd rather have an error than actually going to prod. But, probably a cleaner error than an arbitrary `KeyError`.